benchmarks/tensorexpr/__main__.py

```diff
@@ -4,14 +4,16 @@
 import os
 from . import tensor_engine
 
-# import normalization
-
-# import reduction
-
-# import softmax
-# import pooling
-# import conv
-# import matmul
+from . import attention      # noqa: F401
+from . import broadcast      # noqa: F401
```

Comment:
I think only "broadcast" and "elementwise" work for now. But if nothing is causing problems so far, I am fine with that.

Reply:
Some other work too, but some of them don't - you are right. If we don't run them, then don't make any troubles AFAIU.